### ISSUE TITLE
Add option to emit the file names of generated files

### ISSUE THREAD

**Issue by swaldhoer** [gcovr/gcovr#518]
Before gcovr 5 it the output file names gcovr creates were known and the full path to each file could easily be constructed.

Starting with gcovr these files are now using an infix (which is basically a very good thing).

However, it would be good to have an option that tells gcovr to emit the list of files it produces/tries to produce, so that one can consume this output, parse it and use this list as input for later steps in e.g., the build process.

Some sort of globbing is not a solution, because there could be files from old runs, and then files could be passed to the next steps that are not meant to be passed further.

**Comment by Spacetown** [comment]
Are you talking about the detailed files of the html output?

**Comment by swaldhoer** [comment]
Yes, the files are now something like ``index.dummy.c.{SOME_ARBITRARY_NUMBER}.html`` (before 5.0, the filename was just ``index.dummy.c.html``) and therefore it can no longer be known in advance.

**Comment by Spacetown** [comment]
This was done because of too long paths and forbidden characters in windows, see #375. In the past the filename could also contain directories separated by #.
I think now the names are deterministic and it's also possible to have the same filename in different directories.
Why do you need to know the filenames in advance?
